### PR TITLE
Fix in Multi-Language URLs

### PR DESCRIPTION
**Pull request by DevKang**
Fixed problem in URL creation in Not-English Language(in my case Korean)

### CODE REVIEW

**Comment by poolqf** [comment]
Could you add a test to cover it? I would like to know where's the error 😄

**Comment by DevKang** [comment]
if text contains string like "http://url.contains.other.language/한글/", then URL(string: url) returns nil.
So I added percent encoding to resolve this problem

**Comment by poolqf** [comment]
All right, could you add a test please?

**Comment by DevKang** [comment]
Sorry.. It is my first contribution in github. did you meant this test code in project?

File: ActiveLabelTests/ActiveTypeTests.swift [hunk 1]
```diff
@@ -195,6 +195,11 @@ class ActiveTypeTests: XCTestCase {
         XCTAssertEqual(activeElements.count, 1)
         XCTAssertEqual(currentElementString, "pic.twitter.com/YUGdEbUx")
         XCTAssertEqual(currentElementType, ActiveType.url)
+        
+        label.text = "http://url.with.other.language/한글"
+        XCTAssertEqual(activeElements.count, 1)
+        XCTAssertEqual(currentElementString, "http://url.with.other.language/한글")
+        XCTAssertEqual(currentElementType, ActiveType.url)
```
Comment (poolqf) [inline ActiveLabelTests/ActiveTypeTests.swift]: Well, since the change you made is not related with this, what you should add is a test checking that the string you receive when you force a touch on the label, is the correct one, as URL

**Comment by poolqf** [comment]
Yes @DevKang , I've added a comment on your latest commit. If you have any doubt, please tell me, I'll be happy to help 😄

File: ActiveLabelTests/ActiveTypeTests.swift (hunk 1 above)
Comment (DevKang) [inline ActiveLabelTests/ActiveTypeTests.swift]: Thanks for commenting :) 
But I can`t find way to test force touch on label...
Can you help with this??

**Comment by maziyarpanahi** [comment]
The problem was force touch? I am assuming with Apple shifting its focus away from force touch we can assume it really doesn't matter? Personally I never used force touch in my apps, so I don't even know where to begin to test it.
Anyone has a small snippet so I can test on my device and merge this pull request? (after 2 years still passed all the checks!)

**Comment by tralf** [comment]
@maziyarpanahi I created a new PR for this problem with more detailed explanation and unit tests. Please, take a look if you have a chance.

**Comment by maziyarpanahi** [comment]
in favor of https://github.com/optonaut/ActiveLabel.swift/pull/303
